health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -3250,6 +3435,13 @@
 function updateBoundaryData(boundaryData: any[], hierarchy: any[]): any[] {
   const map: Map<string, string> = new Map();
   const count: Map<string, number> = new Map();
+  boundaryData = boundaryData.map(row =>
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)